### ISSUE TITLE
Installation on Windows 8.1: Error: Command failed (65535). 

### ISSUE THREAD

**Issue by grishagin** [hadley/devtools#857]
I have an issue with installation on Windows 8.1.
Followed instructions: 
1) installed Rtools32.exe (tried Rtools33.exe as well) 
2) Path begins like this:
C:\Rtools\bin;
C:\Rtools\gcc-4.6.3\bin;
C:\R\R-3.1.3\bin\x64;
...

Code with error and traceback is below. I googled for hours, and checked everything, performed clean installations of R, Rtools, with and without cygwin, tried different versions and combinations.
What am I missing?
Thank you.
```R
R version 3.1.3 (2015-03-09) -- "Smooth Sidewalk"
Copyright (C) 2015 The R Foundation for Statistical Computing
Platform: x86_64-w64-mingw32/x64 (64-bit)

> require(devtools)
Loading required package: devtools
> has_devel()
"C:/R/R-31~1.3/bin/x64/R" --no-site-file --no-environ --no-save --no-restore CMD SHLIB foo.c 

Error: Command failed (65535)
> traceback()
6: stop("Command failed (", status, ")", call. = FALSE)
5: system_check(r_path, options, c(r_profile(), r_env_vars(), env_vars), 
       ...)
4: force(code)
3: in_dir(path, system_check(r_path, options, c(r_profile(), r_env_vars(), 
       env_vars), ...))
2: R("CMD SHLIB foo.c", tempdir())
1: has_devel()
```

**Comment by grishagin** [comment]
I've tried to look at the code in more detail, and here's what I found: foo.dll cannot be found. It is indeed not there, or anywhere else for that matter. 

```R
> has_devel
function () 
{
    foo_path <- file.path(tempdir(), "foo.c")
    cat("void foo(int *bar) { *bar=1; }\n", file = foo_path)
    on.exit(unlink(foo_path))
    R("CMD SHLIB foo.c", tempdir())
    dylib <- file.path(tempdir(), paste("foo", .Platform$dynlib.ext, 
        sep = ""))
    on.exit(unlink(dylib), add = TRUE)
    dll <- dyn.load(dylib)
    on.exit(dyn.unload(dylib), add = TRUE)
    stopifnot(.C(dll$foo, 0L)[[1]] == 1L)
    TRUE
}
<environment: namespace:devtools>
> foo_path <- file.path(tempdir(), "foo.c")
> cat("void foo(int *bar) { *bar=1; }\n", file = foo_path)
> foo_path
[1] "C:\\Users\\GRISHA~1\\AppData\\Local\\Temp\\Rtmpi6gTQo/foo.c"
> on.exit(unlink(foo_path))
> dylib <- file.path(tempdir(), paste("foo", .Platform$dynlib.ext, 
+                                     sep = ""))
> dylib
[1] "C:\\Users\\GRISHA~1\\AppData\\Local\\Temp\\Rtmpi6gTQo/foo.dll"
> on.exit(unlink(dylib), add = TRUE)
> dll <- dyn.load(dylib)
Error in inDL(x, as.logical(local), as.logical(now), ...) : 
  unable to load shared object 'C:/Users/GRISHA~1/AppData/Local/Temp/Rtmpi6gTQo/foo.dll':
  LoadLibrary failure:  The specified module could not be found.
```

**Comment by hadley** [comment]
Duplicate of #779 - I'm not sure what the problem is, but I don't think it's devtools related.

**Comment by YanYanEr** [comment]
@grishagin Hello, have you solved this problem? Now I meet the same error, and have not find the solution yet. Would you like to help me if you have solved it? 

Thank you.

**Comment by grishagin** [comment]
Yes, the problem has been resolved, but I am not sure what was the culprit, or the solution for that matter! I simply quit trying for a while after I've exhausted all options. 
The next time I tried to use devtools - it worked, but I already had updated to Windows 10 and had a different version of R. 
Bottomline, it seems to be related to R talking to cmd in windows, and under certain specific circumstances unbeknownst to me, this process fails.
